it-android/build.gradle

```diff
@@ -0,0 +1,68 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+        mavenLocal()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:8.9.3'
+    }
+}
+
+plugins {
+    id 'rhino.spotless-conventions'
+}
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+spotless {
+    java {
+        target "src/*/java/**/*.java"
+    }
+}
+
+
+// read the (android specific) local.properties file
+def localPropertiesFile = rootProject.file("local.properties");
+Properties localProperties = new Properties()
+if (localPropertiesFile.exists()) {
+    localProperties.load(localPropertiesFile.newDataInputStream())
+}
+
+if (!System.getenv("ANDROID_HOME") && !localProperties.get("sdk.dir")) {
+    System.out.println("No Android SDK found. Skipping build")
+} else {
+    apply plugin: 'com.android.application'
+
+    android {
+        namespace = "org.mozilla.javascript.android"
+
+        compileSdkVersion 33
+        defaultConfig {
+            minSdk = 33
+            targetSdk = 33
```

Comment:
Tried to use the lowest possible sdk version, so that the apk is compatible with as many as possible phones, but `./gradlew :it-android:lintDebug` means:

Error: Google Play requires that apps target API level 33 or higher. [ExpiredTargetSdkVersion]

So let's use 33 here